js/RecentDonations.js

```diff
@@ -0,0 +1,63 @@
+
+var app = angular.module('RecentDonations', []);
+app.controller('RecentDonationsControl', function($scope) {
+    
+
+    GLOBALS.GET_PARTICIPANT_INFO(160204, function(data) {
+    	$scope.displayName = data["firstName"]
```

Comment:
I should probably say $scope.$apply here... it's only working now because the next callback takes longer!